build.sbt

```diff
@@ -96,12 +92,12 @@ releaseProcess := Seq[ReleaseStep](
   checkSnapshotDependencies,
   inquireVersions,
   runClean,
-  releaseStepCommandAndRemaining("+test"),
+  runTest,
```

Comment:
Does this run tests over all versions?

Reply:
Well spotted! sbt-release added built-in support for [cross building and cross publishing](https://github.com/sbt/sbt-release#cross-building-during-a-release) since version 0.7, but we need to set `releaseCrossBuild` to true, which I forgot. Added it in 642f4fb.